status-im/status-react
# added SPANK token

NEW:
this PR now also fixes #5387 

This PR includes #5207 and two additions:

- adds the icon files to resources
- fixes the token name

Super big thanks and kudos to @ChaceHunter on whose original PR this was based.

### Summary:

Adds SPANK token.
Fixes the bug with checksummed contract addresses

#### Expected behavior
All tokens show up in transaction history when user sends or receives them.

#### Actual behavior
Some tokens (SPANK in that PR and also PRG, XAUR, STX)

### Steps to test:
- Open Status in mainnet
- Get and enable SPANK and any of these tokens 
- Send and receive them and check transaction history


status: ready
